src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -671,16 +671,32 @@ export function useUsersValidation(): UsersStepValidation {
   }
 
   for (let index = 0; index < users.length; index++) {
-    const invalidGroups = [];
-    const userNameError = validateUserName(users, users[index].name, index);
-    const sshKeyError = validateSshKey(users[index].ssh_key);
+    const userErrors: { [key: string]: string } = {};
+    const isUserDefined =
+      !!users[index].password ||
+      !!users[index].ssh_key ||
+      users[index].groups.length > 0;
+    if (users[index].name || isUserDefined) {
+      const userNameError = validateUserName(users, users[index].name, index);
+      if (userNameError) {
+        userErrors.userName = userNameError;
+      }
+    }
+
     const isPasswordValid = checkPasswordValidity(
       users[index].password,
       environments.includes('azure'),
     ).isValid;
-    const passwordError =
-      users[index].password && !isPasswordValid ? 'Invalid password' : '';
+    if (users[index].password && !isPasswordValid) {
+      userErrors.userPassword = 'Invalid password';
+    }
+
+    const sshKeyError = validateSshKey(users[index].ssh_key);
```

Comment:
I know this isn't really relevant for this PR. I would really like to take a look at how we're handling validation
in the frontend and maybe streamline this a bit :)

Reply:
Yes!!! We now have more places for ssh keys and secrets. This is very similar for CAs. For SSH keys and CAs, I would like to see a pattern of accepting almost anything, but raising a warning. And I really think that having errors as well as warning can be useful in our validation.